frontend/src/Models.js

```diff
@@ -0,0 +1,21 @@
+import React, { useEffect, useState } from "react";
+
+const ModelsContext = React.createContext({
+  models: [], fetchModels: () => {}
+})
+export default function Models(){
+  const [models, setModels] = useState([])
+    const fetchModels = async () => {
+      const response = await fetch("http://localhost:8000/")
```

Comment:
Could use a config variable here instead of localhost to avoid losing track of the requests